lms/envs/common.py

```diff
@@ -1330,15 +1330,17 @@
     },
     'style-certificates': {
         'source_filenames': [
-            'certificates/css/main-ltr.css',
+            'sass/certificates/main-ltr.css',
+            'css/vendor/font-awesome.css',
         ],
-        'output_filename': 'css/certificates-style.css',
+        'output_filename': 'css/certificates-style.css'
     },
     'style-certificates-rtl': {
         'source_filenames': [
-            'certificates/css/main-rtl.css',
+            'sass/certificates/main-rtl.css',
```

Comment:
@talbs it sounds like there is something that you can't account for when you compile locally that gets fixed if you compiled the sass files within the context of the lms.

@Kelketek the original plan by @talbs is to have a checked in version of the CSS that is generated by a UX team member and only have the sass here as a reference.  I don't have a strong opinion on which is the "right" way of doing this.  If we are going to check in the css, we should reference it directly.  If we are going to check in the sass and have it compile at runtime, then your changes make sense.n